templates/todo/tag_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Tags</h1>
+    <button style="float: right; margin-right: 25px; bottom: 50px; position:relative;" class="btn btn-primary">
```

Comment:
Remove styles from here and add to styles.css this:

.btn-tags{
float: right; 
margin-right: 25px; 
bottom: 50px; 
position:relative;
}

```suggestion
    <button  class="btn btn-primary btn-tags">
```